libr/anal/cc.c

```diff
@@ -211,6 +211,30 @@ R_API void r_anal_cc_set_error(RAnal *anal, const char *convention, const char *
 	r_strbuf_fini (&sb);
 }
 
+#if R2_580
```

Comment:
typedef struct {
	void *db;
	 char *cc;
	 int oldArg;
} RAnalCCMaxCache;

Reply:
store this into anal->ccmax_cache

Reply:
```
static bool max_cc_cached (RAnal *anal, const char *cc, void *db) {
  return (anal->ccmax_cache.db == db && anal->ccmax_cache.oldCC && anal->ccmax_cache.oldCC.cc == cc);
}
```

Reply:
Keeping the structs similar as https://github.com/radareorg/radare2/pull/20443